WordPress/Classes/ViewRelated/Pages/EditPageViewController.m

```diff
@@ -50,4 +51,14 @@ - (AbstractPost *)createNewDraftForBlog:(Blog *)blog {
     return [PostService createDraftPageInMainContextForBlog:blog];
 }
 
+#pragma mark - Onboarding
+
+- (void)showOnboardingTips
+{
+    CGFloat xValue = IS_IPAD ? CGRectGetMaxX(self.view.frame)-NavigationBarButtonRect.size.width-20.0 : CGRectGetMaxX(self.view.frame)-NavigationBarButtonRect.size.width-10.0;
```

Comment:
PS: I know this code is from before your changes though.  But it could be a nice addition.